google/api_core/client_info.py

```diff
@@ -64,13 +65,15 @@ def __init__(
         gapic_version=None,
         client_library_version=None,
         user_agent=None,
+        rest_version=None,
     ):
         self.python_version = python_version
         self.grpc_version = grpc_version
         self.api_core_version = api_core_version
         self.gapic_version = gapic_version
         self.client_library_version = client_library_version
         self.user_agent = user_agent
+        self.rest_version = rest_version
```

Comment:
Should there be a check that either `rest_version` XOR `grpc_version` is set?

Reply:
Sounds reasonable to me, I was thinking about the same thing, but then decided that it would be safer to not add something like tha to a GA'ed library. If you feel strong about it, I can add it. Should I?

Reply:
I don't feel _strongly_ about it, but it seems like a reasonable thing for someone looking at the fields to expect.